### PR TITLE
Introducing Dapper integrations

### PR DESCRIPTION
**Pull request by Paul-Brustinov**

### CODE REVIEW

File: SD/Models/IssueModel.cs [hunk 1]
```diff
@@ -7,15 +8,28 @@ public class IssueModel
     {
         public int Id { get; set; }
 
-        public string Name{get; set;}
+        public string Theme{get; set;}
```
Comment (haswelldev) [inline SD/Models/IssueModel.cs]: topic?

File: SD/Models/IssueModel.cs [hunk 2]
```diff
@@ -7,15 +8,28 @@ public class IssueModel
...
 
-        public string Memo { get; set; }
+        public string Matter { get; set; }
```
Comment (haswelldev) [inline SD/Models/IssueModel.cs]: what is it?
reason?
description?

File: SD/Program.cs [hunk 3]
```diff
@@ -1,12 +1,21 @@
...
 
 namespace SD
 {
     public class Program
     {
         public static void Main(string[] args)
         {
+            new DescStorage().Agents.ToList().ForEach(a=> Console.WriteLine(string.Format("{0}, {1}", a.Name, a.Login)));
+            new DescStorage().Companies.ToList().ForEach(c=> Console.WriteLine(string.Format("{0}", c.Name)));
+            new DescStorage().Issues.ToList().ForEach(r=> Console.WriteLine(string.Format("{0}", r.Theme, r.Matter)));
```
Comment (haswelldev) [inline SD/Program.cs]: clear main

File: SD/web.config [hunk 4]
```diff
@@ -11,4 +11,10 @@
     </handlers>
     <aspNetCore processPath="%LAUNCHER_PATH%" arguments="%LAUNCHER_ARGS%" stdoutLogEnabled="false" stdoutLogFile=".\logs\stdout" forwardWindowsAuthToken="false"/>
   </system.webServer>
+
+  <connectionStrings>
+    <add name="2x2crmDb"
+         connectionString="Server=jdbc:mysql://localhost/;Port=3306;Database=2x2crm;Uid=root;Pwd=123;" />
```
Comment (haswelldev) [inline SD/web.config]: jdbc? 0_0

File: SD/Storage/DescStorage.cs [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+using SD.Storage;
+using MySql.Data.MySqlClient;
+using Dapper;
+
+namespace service_desk.Storage
+{
+    public class DescStorage : IServiceDeskStorage
+    {
+
+        private MySqlConnection connection;
```
Comment (haswelldev) [inline SD/Storage/DescStorage.cs]: private field's name should begin with underscore
`_connection`

File: SD/Models/IssueModel.cs [hunk 6]
```diff
@@ -1,4 +1,5 @@
 ﻿using System.Collections.Generic;
+using service_desk.Models;
```
Comment (haswelldev) [inline SD/Models/IssueModel.cs]: fix namespaces
it should be SD everywhere

File: SD/Models/CompanyModel.cs [hunk 7]
```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace service_desk.Models
```
Comment (haswelldev) [inline SD/Models/CompanyModel.cs]: SD

File: SD/Controllers/IssueController.cs [hunk 8]
```diff
@@ -8,6 +8,8 @@ public class IssueController : Controller
         [HttpGet]
         public IActionResult GetAll()
         {
+
+
```
Comment (haswelldev) [inline SD/Controllers/IssueController.cs]: return collection of issues here

**Review by haswelldev — CHANGES_REQUESTED** [review]
Also please add README.md file with description how to setup and wipe database.

File: SD/Models/IssueModel.cs (hunk 2 above)
Comment (Paul-Brustinov) [reply SD/Models/IssueModel.cs]: Matter is a long string description of the issue.

File: SD/Program.cs (hunk 3 above)
Comment (Paul-Brustinov) [reply SD/Program.cs]: Where can I test how work storage?

File: SD/web.config (hunk 4 above)
Comment (Paul-Brustinov) [reply SD/web.config]: Why not? What should be in connection string?

File: SD/Program.cs (hunk 3 above)
Comment (haswelldev) [reply SD/Program.cs]: @Paul-Brustinov you should remove your "test code" before merging

File: SD/web.config (hunk 4 above)
Comment (Paul-Brustinov) [reply SD/web.config]: In fact this connectionString I can delete, it is not used.

Comment (haswelldev) [reply SD/web.config]: @Paul-Brustinov 
jdbc means java database connectivity

File: SD/Program.cs (hunk 3 above)
Comment (Paul-Brustinov) [reply SD/Program.cs]: I need something to store this code... esle I can lose it

Comment (haswelldev) [reply SD/Program.cs]: let it go

**Review by haswelldev — APPROVED** [review]

**Pull request merged**